scipy-stubs/stats/_distribution_infrastructure.pyi

```diff
@@ -0,0 +1,448 @@
+        *,
+        validation_policy: _ValidationPolicy = None,
+        **params: onp.ToFloat | onp.ToFloatND,
+    ) -> None: ...
+
+    #
+    def __neg__(self, /) -> ShiftedScaledDistribution[Self, _FloatingT_co, _ShapeT0_co]: ...
+    def __abs__(self, /) -> FoldedDistribution[Self, _FloatingT_co, _ShapeT0_co]: ...
+
+    # TODO(jorenham): Accept `onp.ToFloatND`?
+    def __add__(self, rshift: onp.ToFloat, /) -> ShiftedScaledDistribution[Self, _FloatingT_co, _ShapeT0_co]: ...
+    def __sub__(self, lshift: onp.ToFloat, /) -> ShiftedScaledDistribution[Self, _FloatingT_co, _ShapeT0_co]: ...
+    def __mul__(self, scale: onp.ToFloat, /) -> ShiftedScaledDistribution[Self, _FloatingT_co, _ShapeT0_co]: ...
+    def __truediv__(self, iscale: onp.ToFloat, /) -> ShiftedScaledDistribution[Self, _FloatingT_co, _ShapeT0_co]: ...
+    def __pow__(self, exp: onp.ToInt, /) -> MonotonicTransformedDistribution[Self, _FloatingT_co, _ShapeT0_co]: ...
+    __radd__ = __add__
+    __rsub__ = __sub__
+    __rmul__ = __mul__
+    __rtruediv__ = __truediv__
+    __rpow__ = __pow__
+
+    #
+    def reset_cache(self, /) -> None: ...
+
+    #
+    def plot(
+        self,
+        x: _PlotQuantity = "x",
+        y: _PlotQuantity = "pdf",
+        *,
+        t: tuple[_PlotQuantity, _JustFloat, _JustFloat] = ("cdf", 0.0005, 0.9995),
+        ax: _AxesT | None = None,
+    ) -> _AxesT: ...
+
+    #
+    # TODO: This will be removed after 1.15.0rc1 (1.15.0rc2 perhaps?): https://github.com/scipy/scipy/pull/22149
+    @overload
+    def llf(self, sample: onp.ToFloat | onp.ToFloatND, /, *, axis: None) -> _Float: ...
+    @overload
+    def llf(self: _CDist0, sample: onp.ToFloat | onp.ToFloatStrict1D, /, *, axis: AnyShape | None = -1) -> _Float: ...
+    @overload
+    def llf(
+        self: _CDist[_ShapeT1], sample: onp.ToFloat | onp.ToFloatStrict1D, /, *, axis: AnyShape = -1
+    ) -> onp.ArrayND[_Float, _ShapeT1]: ...
+    @overload
+    def llf(
+        self: _CDist0, sample: onp.ToFloatStrict2D, /, *, axis: op.CanIndex | tuple[op.CanIndex] = -1
+    ) -> onp.Array1D[_Float]: ...
+    @overload
+    def llf(self: _CDist0, sample: onp.ToFloatStrict2D, /, *, axis: tuple[op.CanIndex, op.CanIndex]) -> _Float: ...
+    @overload
+    def llf(
+        self: _CDist0, sample: onp.ToFloatStrict3D, /, *, axis: op.CanIndex | tuple[op.CanIndex] = -1
+    ) -> onp.Array2D[_Float]: ...
+    @overload
+    def llf(self: _CDist0, sample: onp.ToFloatStrict3D, /, *, axis: tuple[op.CanIndex, op.CanIndex]) -> onp.Array1D[_Float]: ...
+    @overload
+    def llf(self: _CDist0, sample: onp.ToFloatStrict3D, /, *, axis: tuple[op.CanIndex, op.CanIndex, op.CanIndex]) -> _Float: ...
+    @overload
+    def llf(
+        self: _CDist[onp.AtLeast1D], sample: onp.ToFloat | onp.ToFloatND, /, *, axis: AnyShape = -1
+    ) -> onp.ArrayND[_Float]: ...
+    @overload
+    def llf(self, sample: onp.ToFloat | onp.ToFloatND, /, *, axis: AnyShape | None = -1) -> _Float | onp.ArrayND[_Float]: ...
```

Comment:
Note to self: remove this if there'll be no `rc2`, see https://github.com/scipy/scipy/pull/22149

Reply:
there'll be an `rc2` 🎉